examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,30 +62,29 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-clf = make_pipeline(Xdawn(n_components=3),
-                    EpochsVectorizer(),
+X = EpochsVectorizer().fit_transform(epochs)
+labels = epochs.events[:, -1]
+clf = make_pipeline(XdawnTransformer(n_components=3,
+                    n_chan=epochs.info['nchan']),
                     MinMaxScaler(),
                     LogisticRegression(penalty='l1'))
```

Comment:
Can you put the `Vectorizer` after `MinMaxScaler()`, and replace `X = EpochsVectorizer().fit_transform(epochs)` with `X = epochs.get_data()` since we now agreed that `X` can be multidim?